packages/core/src/openapi/openapi-request-builder.ts

```diff
@@ -45,11 +47,11 @@ export class OpenApiRequestBuilder<ApiT, FnT extends keyof ApiT> {
 
   /**
    * Execute request.
-   * @param destination  Destination to execute the request against.
+   * @param destination Destination to execute the request against.
    * @param options Options to employ when fetching destinations.
-   * @returns A promise resolving to the requested return type.
+   * @returns A promise resolving to an AxiosResponse.
    */
-  async execute(
+  async executeRaw(
```

Comment:
Also add a ticket a quickly adjust the docs to reflect the new method on the openApi client.

Reply:
Wait the method is not exported. Is it supposed to be used just internally?